arm-customer-insights/2017-01-01/swagger/customer-insights.json

```diff
@@ -3937,6 +4036,7 @@
         "calculationWindow": {
           "type": "string",
           "enum": [
+            "Lifetime",
```

Comment:
breaking change: this is a property of a type used in responses. So if the server now sends back `Lifetime` here, old clients will throw on deserialization.